<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1360</width>
    <height>748</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestion des Clients</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>851</width>
      <height>601</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../1x/Untitled-1-100.jpg</pixmap>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>100</y>
      <width>821</width>
      <height>461</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Ajouter voiture</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>361</width>
        <height>221</height>
       </rect>
      </property>
      <property name="title">
       <string>Ajouter voiture</string>
      </property>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>47</width>
         <height>13</height>
        </rect>
       </property>
       <property name="text">
        <string>Nom</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>80</y>
         <width>47</width>
         <height>13</height>
        </rect>
       </property>
       <property name="text">
        <string>Modele</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>120</y>
         <width>81</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Categorie</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>160</y>
         <width>81</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Matricule</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="NOM">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>40</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="Categorie">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>120</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="Matricule">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>160</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pb_ajouter">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>180</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Ajouter</string>
       </property>
      </widget>
      <widget class="QComboBox" name="Modele">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>80</y>
         <width>111</width>
         <height>22</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>RENAULT</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>BMW</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>OPEL</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Citroen</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>PEUGEOT</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Autre</string>
        </property>
       </item>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>Afficher voiture</string>
     </attribute>
     <widget class="QTableView" name="tab_voiture">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>441</width>
        <height>231</height>
       </rect>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_5">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>150</y>
        <width>241</width>
        <height>101</height>
       </rect>
      </property>
      <property name="title">
       <string>Trier par Nom</string>
      </property>
      <widget class="QRadioButton" name="radioButton_croissant">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>97</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Tri de A-Z</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioButton_dcroissant">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>70</y>
         <width>97</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Tri de Z-A</string>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="label_25">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>60</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Matricule</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="rech_mat">
      <property name="geometry">
       <rect>
        <x>640</x>
        <y>60</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="rechercher_button">
      <property name="geometry">
       <rect>
        <x>700</x>
        <y>100</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Rechercher</string>
      </property>
     </widget>
     <widget class="Line" name="line">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>270</y>
        <width>231</width>
        <height>20</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="statsmed">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>330</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Consulter</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_26">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>300</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Concultez les statistiques</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_imp">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>360</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>imprimer</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_export">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>360</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Export EX</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_27">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>310</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Exporter l'affichage</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Supprimer voiture</string>
     </attribute>
     <widget class="QFrame" name="frame">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>431</width>
        <height>261</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>80</y>
         <width>56</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Matricule</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="Matricule_supp">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>80</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pb_supprimer">
       <property name="geometry">
        <rect>
         <x>300</x>
         <y>210</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Supprimer</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Modifier voiture</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>50</y>
        <width>401</width>
        <height>211</height>
       </rect>
      </property>
      <property name="title">
       <string>modifier</string>
      </property>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>47</width>
         <height>13</height>
        </rect>
       </property>
       <property name="text">
        <string>Nom</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>80</y>
         <width>47</width>
         <height>13</height>
        </rect>
       </property>
       <property name="text">
        <string>Modele</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>120</y>
         <width>81</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Categorie</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="NOM_2">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>40</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="Modele_2">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>80</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="Categorie_2">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>120</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pb_modifier">
       <property name="geometry">
        <rect>
         <x>310</x>
         <y>180</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Modifier</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_14">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>160</y>
         <width>81</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Matricule</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="Matricule_2">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>160</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_5">
     <attribute name="title">
      <string>Exit</string>
     </attribute>
     <widget class="QPushButton" name="pb_exit">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>180</y>
        <width>191</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>EXIT</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1360</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
